src/sonic-yang-models/yang-models/sonic-hostapd.yang

```diff
@@ -0,0 +1,42 @@
+module sonic-hostapd {
+    namespace "http://github.com/sonic-net/sonic-hostapd";
+    prefix shostapd;
+    yang-version 1.1;
+
+    description
+        "SONiC HOSTAPD";
+
+    revision 2023-08-02 {
+        description "Initial revision.";
+    }
+
+    container sonic-hostapd {
+        description "HOSTAPD top level container.";
+
+        container HOSTAPD_GLOBAL_CONFIG {
+            description
+                "Container for hostapd global config.";
+
+            list HOSTAPD_GLOBAL_CONFIG_LIST {
+                key "global";
```

Comment:
Follow the below approach for this global table as well.

https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-lldp.yang#L49